chapter-2.md

```diff
@@ -741,6 +741,37 @@ test "precision" {
 }
 ```
 
+# Advanced Function Invocation
+Zig offers a `@call` builtin function which gives you more flexibility with which how a function is
```

Comment:
you can shorten this to what `@call` does and to me its more about controlling what the compiler does. :)